content/en/people/index.adoc

```diff
@@ -0,0 +1,27 @@
+---
+title: "People"
+description: "ACL Special Interest Group on Turkic Languages People"
+date: 2020-08-27T19:25:12+02:00
+lastmod: 2020-08-27T19:25:12+02:00
+draft: false
+images: []
+contributors: []
+---
+
+President:: Duygu Ataman mailto:sigturk-president@gmail.com[(email)]
+
+Secretary:: Sardana Ivanova mailto:sigturk-secretary@gmail.com[(email)]
+
+
+== Advisory Committee
```

Comment:
Is this line necessary given that the section is (still) empty?

Reply:
We do not have Advisory Committee yet (similar story for non-org non-officer contributors), but request on issue was to prep these sections